Content updates (#79)
microsoft/azure-spring-cloud-training: 06-build-a-reactive-spring-boot-microservice-using-cosmosdb/README.md

- Click on the "Data Explorer" menu item
  - Expand the container named `azure-spring-cloud-cosmosdb`.
  - In that container, expand the container named `City`.

>   - Click on "Items" and use the "New Item" button to create **these exact sample items**:

Reviewer: IMHO, the bold font comes across as a bit aggressive. This phrasing also raises questions, as to why exactly the sample data can't be changed.

I would suggest...
```suggestion
  - Click on "Items" and use the "New Item" button to create some sample items using the contents below:
```
